section-engineering-education/engineering-education
# Improving Skills List with Icons

### What This PR Does

- Uses the [Devicons](https://devicon.dev/) icon library to replace the skills list (`.Params.skills`) with icons.
- Uses `aria-labels` for accessibility and `rems` for icon sizing

### Files Edited

- **Single Author Template Layout** - `/layouts/authors/single.html`
- **Louise Findlay's Author Profile** - `/content/authors/louise-findlay/index.md`

### How to Test

1. Checkout this PR branch
2. Start the local hugo server - `hugo server -D`
3. Navigate to [localhost:1313/authors/louise-findlay](http://localhost:1313/authors/louise-findlay)
4. Verify that the Skills section shown matches the screenshot below and that the Skills section is still absent on authors' front-matter without the correct metadata

### Screenshot

![EngEd Author Skills Section with Icons](https://user-images.githubusercontent.com/26024131/158015164-b7fe16e2-87e4-400f-8c62-6a0e7e84847d.png)

### Potential Improvements

- Move the in-line CSS style for the `.skills` class to a CSS file - @hborrelli1, would the `/assets/scss/compentents/_author-blocks.scss` file be the most appropriate?
- Add some spacing between the icons and the social media links below

### Related Issue

This closes #7087.
